sympy/sets/sets.py

```diff
@@ -1968,6 +1968,14 @@ def __lt__(self, other):
             raise TypeError("Invalid comparison of set with %s" % func_name(other))
         return self.is_proper_subset(other)
 
+    def __eq__(self, other):
```

Comment:
I have added this because the old test was failing.
`FiniteSet(Rational(3, 10), Rational(1, 5)) == FiniteSet(0.3, 0.2)` was giving `False` before adding `__eq__`.
Let me know if this is fine?